addons/project/static/src/views/project_task_kanban/project_task_kanban_dynamic_group_list.js

```diff
@@ -34,13 +34,18 @@ export class ProjectTaskKanbanDynamicGroupList extends KanbanDynamicGroupList {
         return result;
     }
 
-    async createGroup() {
+    async createGroup(groupName, isFolded = false, additionalData = {}) {
```

Comment:
Signature has a bit changed in #115909

Reply:
what do you think about waiting to your PR ? then i will just do the project part

Reply:
#115909 has landed in master ;-)